newIDE/app/src/BehaviorsEditor/index.js

```diff
@@ -702,6 +745,18 @@ const BehaviorsEditor = (props: Props) => {
           eventsFunctionsExtension={eventsFunctionsExtension}
         />
       )}
+
+      {!!selectedQuickCustomizationPropertiesBehavior && (
+        <QuickCustomizationPropertiesVisibilityDialog
+          open={!!selectedQuickCustomizationPropertiesBehavior}
+          onClose={() => setSelectedQuickCustomizationPropertiesBehavior(null)}
+          propertyNames={selectedQuickCustomizationPropertiesBehavior
```

Comment:
for some reason, passing the gdMapStringPropertyDescriptor to that dialog was not working properly. when getting the keys in the dialog, the result was not the one excepted. I couldn't figure out what was wrong & ended up passing the names directly from there.